src/store/products/index.js

```diff
@@ -12,29 +7,16 @@ let initialState = {
         { name: 'Apples', category: 'food', price: .99, inStock: 500 },
         { name: 'Eggs', category: 'food', price: 1.99, inStock: 12 },
         { name: 'Bread', category: 'food', price: 2.39, inStock: 90 },
-    ],
-    activeCategory: ''
+    ]
 };
 
-
-function reducer(state = initialState, action) {
+function productsReducer(state = initialState, action) {
     switch (action.type) {
         case 'SET':
-            return {
-                ...state,
-                activeCategory: action.payload,
-                products: initialState.products.filter(product => product.category === action.payload.name)
-            }
+            return {products: initialState.products.filter(product => product.category === action.payload.name)};
```

Comment:
We will want to add an action to change `inStock` when we add an item to the cart. Because of this, you will need to modify your set action so it doesn't reset stock every time you switch categories.